src/components/FeatureViews/AdminPortal/AdminDashboard.js

```diff
@@ -171,6 +173,47 @@ const AdminDashboard = (props) => {
     { label: "Access Control" },
   ];
   
+  const QUERIES = {
+    "unpaidsessions": gql`query MyQuery {
+      unpaidSessions {
+        paymentList {
+          parent {
+            user {
+              firstName
+              lastName
+            }
+          }
+        }
+        course {
+          startTime
+          endTime
+          hourlyTuition
+        }
+        lastPaidSessionDatetime
+      }
+    }
+    `,
+  }
+
+// console.log(UNPAID_SESSIONS)
+
+ const unpaidSessionsData = useQuery(QUERIES["unpaidsessions"])
+//  console.log(unpaidSessionsData)
+ 
+ 
+ const calculateUnpaidSessions = data => {
+   if(unpaidSessionsData.data) {
```

Comment:
You shouldn't need this. there's the loading variable i just mentioned. so before this, you should have a line of code like `if(loading) <Loading/>` or something similiar